pkg/kubelet/kuberuntime/utils.go

```diff
@@ -0,0 +1,184 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kuberuntime
+
+import (
+	"fmt"
+	"math/rand"
+	"strconv"
+	"strings"
+
+	"github.com/golang/glog"
+	"k8s.io/kubernetes/pkg/api"
+	runtimeApi "k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
+	kubecontainer "k8s.io/kubernetes/pkg/kubelet/container"
+)
+
+const (
+	// containerNamePrefix is used to identify the containers on the node managed by kubelet
+	containerNamePrefix = "k8s-container"
+	// podSandboxNamePrefix is used to identify the pod sandboxes on the node managed by kubelet
+	podSandboxNamePrefix = "k8s-pod"
+
+	// Taken from lmctfy https://github.com/google/lmctfy/blob/master/lmctfy/controllers/cpu_controller.cc
+	minShares     = 2
+	sharesPerCPU  = 1024
+	milliCPUToCPU = 1000
+)
+
+// Converts kubecontainer.ContainerState to runtimeApi.ContainerState
+func toRuntimeContainerState(state kubecontainer.ContainerState) runtimeApi.ContainerState {
```

Comment:
nit: toRuntimeContainerState converts kubecontainer.ContainerState to runtimeApi.ContainerState.

And also the following functions.